### PR TITLE
Update README.md

### LINKED ISSUE #104: Update Readme accordingly with recent updates
Update Readme : 
  

- [ ] #98 
- [ ] #55 Examples

### PR DESCRIPTION
**Pull request by romulocintra**
Close #104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,20 +190,68 @@ We allow users to specify a `fractionalDigits` option that will display the smal
...
+// Example using fractionalDigits
+new Intl.DurationFormat("en", { fractionalDigits: 2 }).format(duration);
+// => 12.34 sec
+
+// Example using fractionalDigits and milliseconds set to `narrow`
+new Intl.DurationFormat("en", {
+  milliseconds: "narrow",
+  fractionalDigits: 2,
+}).format(duration);
+// => 12s 345.60ms
```
Comment (anba) [inline README.md:208]: The current spec draft text uses `Intl.ListFormat`, which means the results will look more like:
`12 seconds, 345 milliseconds, and 600 microseconds` and
`12 seconds, 345ms, and 600 microseconds`.

File: README.md [hunk 2]
```diff
@@ -182,20 +190,68 @@ We allow users to specify a `fractionalDigits` option that will display the smal
...
-      seconds: 'numeric',
-    }).format(Temporal.Duration.from('PT2H40S')); // => 2:00:40
+const duration = {
+  hours: 2,
+  minutes: 46,
+  seconds: 40,
+};
+
+new Intl.DurationFormat("en").format(duration);
+// => 2 hr 46 min 40 sec
```
Comment (anba) [inline README.md:221]: `2 hours, 46 minutes, and 40 seconds` according to the current draft text.

**Review by ben-allen — APPROVED** [review]
Correctly matches behavior of Intl.ListFormat. Current spec doesn't seem to do fractionalDigits correctly, though? Might need to be another issue

**Comment by ryzokuken** [comment]
Let's get back to this once we've finalized all the normative issues?

**Comment by sffc** [comment]
@ben-allen has a new PR in https://github.com/tc39/proposal-intl-duration-format/pull/168 which makes most of these changes; after that PR is merged, Ben will take one final look at this PR and close it if all of its loose ends have been resolved.

**Comment by ryzokuken** [comment]
#168 has been merged.

**Pull request closed without merging**